### PR TITLE
IBM Watson Text to Speech (Independent Publisher)

### PR DESCRIPTION
**Pull request by lucastitus**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Screenshot 2023-02-07 171610

Screenshot 2023-02-07 171338

### CODE REVIEW

**Comment by lucastitus** [comment]
@microsoft-github-policy-service agree

File: independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,311 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "IBM Watson Text to Speech",
+    "description": "The IBM Watson Text to Speech service provides APIs that use IBM's speech-synthesis capabilities to synthesize text into natural-sounding speech in a variety of languages, dialects, and voices.",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json:5]: @lucastitus Please remove the word "API" form the description. Please follow our requirements [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

File: independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        "description": "Synthesizes text to audio that is spoken in the specified voice. The service bases its understanding of the language for the input text on the specified voice. Use a voice that matches the language of the input text.",
+        "operationId": "Synthesize"
+      }
+    },
+    "/v1/voices": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json:137]: @lucastitus empty schema is not allowed, please remove it or define a proper schema.

File: independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+                  "title": "Text"
+                }
+              },
+              "required": [
+                "text"
+              ]
+            }
+          }
+        ],
+        "description": "Synthesizes text to audio that is spoken in the specified voice. The service bases its understanding of the language for the input text on the specified voice. Use a voice that matches the language of the input text.",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json:128]: @lucastitus I would suggest you keep all operations' descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

File: independent-publisher-connectors/IBM Watson Text to Speech/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          "tooltip": "Provide the API Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0F62FE",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/apiProperties.json:50]: @lucastitus for all our independent publishers, we ask to set the iconBrandColor to "#da3b01". Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

File: independent-publisher-connectors/IBM Watson Text to Speech/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "SetHost",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('url')"
+        }
+      }
+    ],
+    "publisher": "Lucas Titus"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/apiProperties.json:64]: @lucastitus please add the stackOwner property. 
"stackOwner": "Name of company or organization that connector is connecting to."

File: independent-publisher-connectors/IBM Watson Text to Speech/readme.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# **IBM Watson Text to Speech**
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/readme.md:1]: @lucastitus please follow and include all required sections from our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md), and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).

File: independent-publisher-connectors/IBM Watson Text to Speech/settings.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/IBM Watson Text to Speech/settings.json:1]: @lucastitus Please remove the setting.json file from this pull request as it is not required. 
Thank you.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by lucastitus** [comment]
Hi @Amjed-Ayoub thanks for this!
I have removed the settings.json file as requested. I had kept it as it references the script.csx file, I just want to make sure that the script will still be used without it? It seems the apiProperties schema doesn't allow the "scriptOperations" parameter either - I have kept it in for now though.

**Comment by Amjed-Ayoub** [comment]
> Hi @Amjed-Ayoub thanks for this! I have removed the settings.json file as requested. I had kept it as it references the script.csx file, I just want to make sure that the script will still be used without it? It seems the apiProperties schema doesn't allow the "scriptOperations" parameter either - I have kept it in for now though.

Yes that is totally okay. our apiProperties validator is not updated to handle the scriptoperations, please ignore this error.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @lucastitus,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @lucastitus,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @lucastitus,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Pull request merged**